cli/dcoscli/node/main.py

```diff
@@ -3,19 +3,24 @@
 Usage:
     dcos node --info
     dcos node [--json]
+    dcos node log [--follow --lines=N --master --slave=<slave-id>]
```

Comment:
One way to get around this is to make `--master` implicit. The main problem I see with this is how do we document it given how we currently structure the `--help` output.